src/views/Pools/components/PoolCard.tsx

```diff
@@ -96,7 +95,7 @@ const PoolCard: React.FC<HarvestProps> = ({ pool }) => {
   const [onPresentCompound] = useModal(
     <CompoundModal earnings={earnings} onConfirm={onStake} tokenName={stakingTokenName} />,
   )
-
+  const poolImage = pool.tokenName.concat('-', pool.stakingTokenName).replace('QuoteToken.', '').toLocaleLowerCase()
```

Comment:
remove .replace('QuoteToken.', '')

Reply:
replace by 
```
const poolImage = `${pool.tokenName}-${pool.stakingTokenName}.svg`.toLocaleLowerCase()
```

and remove the `.svg` line 130

Reply:
done